resources/js/api/api.js

```diff
@@ -27,7 +27,9 @@ export default class API {
         }
 
         return response.json().then(data => {
-            notyf.success(data.data.message);
+            if (data && data.data && data.data.message) {
```

Comment:
So now there is just no notification for `read all notifications`?

Reply:
With this solution: Yes.

We'll have to add translations to vue since it's bad practice to have the API return "static" localized strings. It would be better to implement this in the frontend but I don't know which i18n package to use yet. 

Either we prioritize good api design and ignore the notification for now (probably not a bad idea since the list automatically clears on success. imho there's no notification needed), or we wait until we have i18n sorted out and draft this PR.